doc/source/release-notes/Percona-Server-8.0.28-20.rst

```diff
@@ -0,0 +1,40 @@
+.. _8.0.28-19:
+
+================================================================================
+*Percona Server for MySQL* 8.0.28-20 (2022-06-)
+================================================================================
+
+.. include:: ../_res/rn/ps-mysql-blurb.txt
+
+.. contents::
+   :local:
+
+.. include:: ../_res/rn/8.0.28-highlights.txt
+
+Improvement
+=================================================
+
+* :jirabug:`PS-8103`: Memory leak and errors detectors (Valgrind or AddressSanitizer) provide detailed stack traces from dynamic libraries (plugins and components). The detailed stack traces make it easier to identify and fix the issues.
+* ZenFS file system plugin for RocksDB is updated to version 2.1.0.
+
+New Features
+=================================================
+
+* :jirabug:`PS-7044`: Implements support for encryption user-defined functions (UDFs) for OpenSSL.
+* :jirabug:`PS-7672`: Implements support for the Amazon Key Management Service component in Percona Server for MySQL.
+* :jirabug:`PS-7748`: Implements the ability to log error messages to a memory buffer.
+
+Bugs fixed
+=================================================
+
+* :jirabug:`PS-6029`: Data masking ``gen_rnd_us_phone()`` function had a different format compared to MySQL upstream version.
+* :jirabug:`PS-8136`: ``LOCK TABLES FOR BACKUP`` did not prevent InnoDB key rotation. Due to this behavior, Percona Xtrabackup couldn't fetch the key in case the key was rotated after starting the backup.
+* :jirabug:`PS-8143`: Fixed the memory leak in ``File_query_log::set_rotated_name()``. 
+* :jirabug:`PS-7894`: When a query to the MyRocks table was interrupted due to the ``MAX_EXECUTION time`` option, an incorrect error message was received. (Thanks to user hagrid-the-developer for reporting this issue.)
+* :jirabug:`PS-8158`: There was access to possibly not initialized memory. (Thanks to Rinat Ibragimov for reporting this issue.)
+* :jirabug:`PS-5008`: Fixed the memory leak in ``sync_latch_meta_init()`` after mysqld shutdown.
+* **zenfs** utility failed when a user tried to restore a single file into a specified ZenFS path.
```

Comment:
Potentially, if this is not against any rules, we can add links to WDCZEN-XXX bugs here
https://jira.percona.com/browse/WDCZEN-29
https://jira.percona.com/browse/WDCZEN-31
https://jira.percona.com/browse/WDCZEN-32

Reply:
Sorry. Just saw the part with Gorge's comments about this project being private.
Please ignore my previous message.